src/add.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..200)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? == true || @parent_permission == true
+  end
+
+  private
+
+  def of_age?
+    if @age >= 18
+      true
+    elsif @age < 18
+      false
+    end
```

Comment:
- [ OPTIONAL ] I think it is here also you write it like this, that's right but I recommend refactor it to just be like this

```ruby
def is_of_age?
    @age >= 18
  end
```

Reply:
@Reem-lab Thank you, I already solved it 💯 .